QRCoder/QRCodeGenerator.cs

```diff
@@ -289,17 +291,14 @@ private static QRCodeData GenerateQrCode(BitArray bitArray, ECCLevel eccLevel, i
             ModulePlacer.AddQuietZone(qr);
             return qr;
 
-            void AddCodeWordBlocks(int blockNum, int blocksInGroup, int codewordsInGroup, BitArray bitArray2, int offset2, int count, Polynom generatorPolynom)
+
+            void AddCodeWordBlocks(int blockNum, int blocksInGroup, int codewordsInGroup, int offset2, int count, Polynom generatorPolynom)
             {
                 var groupLength = codewordsInGroup * 8;
                 for (var i = 0; i < blocksInGroup; i++)
                 {
-                    var bitBlockList = BinaryStringToBitBlockByteList(bitArray2, offset2, groupLength);
                     var eccWordList = CalculateECCWords(bitArray2, offset2, groupLength, eccInfo, generatorPolynom);
```

Comment:
Should this be?
```suggestion
                    var eccWordList = CalculateECCWords(bitArray, offset2, groupLength, eccInfo, generatorPolynom);
```

Reply:
Yeah, probably that was the only change needed.

Reply:
I wasn't sure so I reverted the merge just so I could review my prior changes and reapply them.  It's all set now, I believe.  Tests pass locally anyway.

Reply:
Thanks. As soon as the action runner succeeds, I'll merge. :-)